boards/shields/xenvm_dom0/Kconfig.defconfig

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+if SHIELD_XENVM_DOM0
+
+config NUM_IRQS
+	int
+	default 500
+
+config MAX_XLAT_TABLES
+	default 24
+
+config HEAP_MEM_POOL_SIZE
+	default 2097152
+
+# Enable PV hypervisor console
+config UART_XEN_HVC_CONSOLEIO
+	default y
+
+config TEST
+	default y
+
+config SHELL
+	default y
+
+config SHELL_BACKEND_SERIAL
+	default y
+
+config DEBUG_INFO
+	default y
+
+config OVERRIDE_FRAME_POINTER_DEFAULT
+	default y
+
+config OMIT_FRAME_POINTER
+	default y
+
+config MAIN_STACK_SIZE
+	default 32768
+
+config IDLE_STACK_SIZE
+	default 32768
+
+config ISR_STACK_SIZE
+	default 32768
+
+config SYSTEM_WORKQUEUE_STACK_SIZE
+	default 32768
+
+config PRIVILEGED_STACK_SIZE
+	default 32768
+
+config SHELL_STACK_SIZE
+	default 32768
+
+endif # SHIELD_XENVM_DOM0
```

Comment:
I believe that you have valid reason to define all the stack sizes there

Reply:
I think that we need to add stack changes with `TODO:` as they were added to xenvm_defconfig. No one investigated exact value, that we need, it was just a WA for Gen3 boards.